ups/eupspkg.cfg.sh

```diff
@@ -15,7 +15,7 @@ if [[ $OSTYPE == darwin* ]]; then
 	# with / (absolute) or @ (in which case it's most likely a @rpath).
 	# Note that $LIBPYTHON_DYLIB may not exist (e.g., if Python lib has
 	# been built as a framework -- like /usr/bin/python)
-	LIBPYTHON_DYLIB=$(python -c "import sysconfig, os.path; print os.path.join(*sysconfig.get_config_vars('LIBDIR', 'LDLIBRARY'))")
+	LIBPYTHON_DYLIB=$(python -c "from __future__ import print_function; import sysconfig, os.path; print(os.path.join(*sysconfig.get_config_vars('LIBDIR', 'LDLIBRARY')))")
```

Comment:
I don't believe you need the print_function import here because you are only printing a single item and python2 allows `print(one_thing)`.